ls1intum/Artemis
# [refactor] removed get call, renamed id in StudentQuestionAnswerResource to answerId

### Checklist

### Steps for Testing


1. Log in to Artemis
2. Create a StudentQuestion
3. Login to at least Tutor, answer the Question
4. Login to initial user, check if Answer is visible
